src/lib/util/VersionUpgradeManager.ts

```diff
@@ -34,6 +34,15 @@ upgradeScriptsByVersion.set('v2.7.0', (config: ConfigContent): Promise<void> =>
 	}
 	return Promise.resolve();
 });
+upgradeScriptsByVersion.set('v3.0.0', (config: ConfigContent): Promise<void> => {
```

Comment:
Would this still apply if someone directly gets a `3.1.0`?

Reply:
Can we have a test case for this scenario as well?

Reply:
There should be no risk of the script firing, but I'll either add a test scenario or point to one that already exists. Update scripts are triggered if the config's `currentVersion` property is null or below the package's `version` property. In that case, all scripts between the old version and the new version are run, and the `currentVersion` property is updated.
For a fresh install, the config doesn't exist at all, so it will use the default config, whose `currentVersion` is derived directly from the package itself, so the scripts don't fire.

Reply:
@rmohan20 , we already have a test to make sure that the right scripts are fired. Please take a look at [line 102 of `VersionUpgradeManager.test.ts`](https://github.com/forcedotcom/sfdx-scanner/blob/0dcfff7b6d7511a5365710dda4d91785ac59af28/test/lib/util/VersionUpgradeManager.test.ts#L102) and let me know if you feel it's sufficient.

Reply:
@jfeingold35 Appears to be good coverage, thanks!